DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UnsupportedViewRecorderTests.swift

```diff
@@ -5,6 +5,7 @@
  */
 
 import XCTest
+#if !os(tvOS)
```

Comment:
Why in some places are we using `#if os(iOS)` and others we're using `#if !os(tvOS)`? Do some of these support macOS?

Reply:
it depends on the context where the if-def is. The one you mentioned indeed should be `#if os(iOS)` because for SR - we don't compile for other than iOS.

Somewhere down for Mocks, which are part of `WebViewTracking`, we have to compile out for tvOS hence `#if os(iOS)`.

This is our existing pattern for `Source` files, I'm trying to apply same for `Test` files.

Reply:
Since it is in SessionReplay it feels like it could be `#if os(iOS)` as in other files 🤔

Reply:
agreed, I reverted it.